### ISSUE TITLE
mutiple level tree

### ISSUE THREAD

**Issue by AkiraEisen** [dalestone/angular2-gantt#30]
if this plugin could provide mutile level tree in left datagrid? e.g. more than 3 level depth.

best regards

**Comment by dalestone** [comment]
Yep I did start this but it required quite a lot of changes to make it work correctly so it has moved to backlog for a future release
